docs/pages/tutorial/deploy-to-mainnet/main.md

```diff
@@ -0,0 +1,172 @@
+
+```sh
+# usage: source .env.sepolia
+export STARKNET_RPC_URL=<RPC_PROVIDER_URL>
+export DOJO_ACCOUNT_ADDRESS=<YOUR_ACCOUNT_ADDRESS>
+export DOJO_PRIVATE_KEY=<YOUR_ACCOUNT_PRIVATE_KEY>
+```
+
+
+
+### Deployment
+
+* Load `.env.sepolia`.
+
+**IF FOR ANY REASON YOU ABORT THE DEPLOYMENT, JUMP TO THE CLEANUP STEP TO UNDO THIS**
+
+```sh
+cd dojo
+source .env.sepolia
+```
+
+* Build and check for errors. If the `migrate plan` command raises any errors, fix and try again before continuing.
+
+```sh
+cd dojo
+sozo -P sepolia clean
+sozo -P sepolia build
+sozo -P sepolia migrate plan
+```
+
+* Execute migration using the [`migrate`](https://github.com/rsodre/512karat/blob/main/dojo/migrate) script...
+
+```sh
+cd dojo
+./migrate sepolia
+```
+
+Your world is deployed!
+
+
+### Cleanup env !!!!!!!
+
+Clear env after all is done...
+
+**THIS IS VERY IMPORTANT, OR YOUR NEXT LOCAL DEPLOYMENT MAY GO TO SEPOLIA OR MAINNET!**
+
+```sh
+cd dojo
+source .env.clear
+```
+
+
+
+### Torii Indexer
+
+Now, if you're building a Dojo client, you will need a Torii service to index our world...
+
+* Install [slot](https://github.com/cartridge-gg/slot) or update it
+
+```sh
+slotup
+```
+
+* Authorize
+
+```sh
+slot auth login
+```
+
+* Find your world starting block. Replace your world address on the following link, clink on **Deployed At Transaction Hash** and write down the **Block Number**. (it may take a while before the link works if you use it right after deployment)
+
+[https://sepolia.starkscan.co/contract/0x04c0970c9f52045ef8eeedd1e11265ebb69ed90fce58c96ad103aecf7f91302a](https://sepolia.starkscan.co/contract/0x04c0970c9f52045ef8eeedd1e11265ebb69ed90fce58c96ad103aecf7f91302a)
+
+* Create Torii service with this command, replacing...
+  * `SERVICE_NAME` can be the name of the game/dapp. Once you create it, you own that name.
+  * `DOJO_VERSION`: your Dojo version (ex: `v1.0.0-alpha.11`)
+  * `WORLD_ADDRESS`: from your Dojo config file [`dojo_sepolia.toml`](https://github.com/rsodre/512karat/blob/main/dojo/dojo_sepolia.toml)
+  * `RPC_URL`: your RPC provider url
+  * `STARTING_BLOCK`: the deployment transaction block we just found before
+  * Take a note of the endpoints after it is deployed...
+
+```sh
+slot deployments create <SERVICE_NAME> torii --version <DOJO_VERSION> --world <WORLD_ADDRESS> --rpc <RPC_URL> --start-block <STARTING_BLOCK> --index-pending true
+```
+
+* slot will output something like this. Save it for later, you will need the endpoints on your client.
+
+```
+Deployment success 🚀
+
+Configuration:
+  World: 0x4c0970c9f52045ef8eeedd1e11265ebb69ed90fce58c96ad103aecf7f91302a
+  RPC: <RPC_PROVIDER_URL>
+  Start Block: 155777
+  Index Pending: false
+
+Endpoints:
+  GRAPHQL: https://api.cartridge.gg/x/512karat-sepolia/torii/graphql
+  GRPC: https://api.cartridge.gg/x/512karat-sepolia/torii
+
+Stream logs with `slot deployments logs 512karat-sepolia torii -f`
```

Comment:
output could be updated to latest slot version too.

Reply:
updated